testing/test_scripts/check_install.sh

```diff
@@ -0,0 +1,19 @@
+###############################################################################
+
+# This script tests the if a perticular tool is installed or not
+
+#Usage:
+# $ sh check_install.sh <tool_name>
+
+################################################################################
+
+#!/bin/bash
+
+if [ `which $1 2>&1 >/dev/null | grep -c no` -eq 1 ]
```

Comment:
How to make sure that it MUST be equal to 1 if it does not exist? It is suggested to use some simpler solution, such as [ -z "$(which $1 2>/dev/null)" ].